src/monda/sorad/qc.py

```diff
@@ -11,11 +11,18 @@
 
 Tom Jordan - tjor@pml.ac.uk - Feb 2022
 """
-
+import sys
 import numpy as np
 import datetime
 import pandas as pd
+from statistics import mode
+
 
+import logging
+log = logging.getLogger('sorad-downloader')
```

Comment:
change this from sorad-downloader to sorad-qc

Reply:
done